src/sonic-config-engine/sonic_device_util.py

```diff
@@ -29,6 +29,39 @@ def get_machine_info():
             machine_vars[tokens[0]] = tokens[1].strip()
     return machine_vars
 
+def get_num_npus():
+   platform = get_platform_info(get_machine_info)
+   num_npus = 1
+   asic_conf_file_path = os.path.join('/usr/share/sonic/device', platform, 'asic.conf')
```

Comment:
i see multiple references to this string. /usr/share/sonic/device, can you make it a constant to use?

Reply:
Fixed